File: packages/pipeline/fixtures/csharp-cross/Controller.cs
namespace Crib.Example;

using Crib.Example.Base;
using Crib.Example.Service;
using Crib.Example.Token;

public class Controller : Base, IGreeter
{
    private Service service;

    public Token Issue(string input)
    {
        return new Token(input);
    }

    public string Greet(string user)
    {
        return service.Greet(user);
    }
}

File: packages/pipeline/fixtures/java-cross/Controller.java
package com.example;

import com.example.Base;
import com.example.Service;
import com.example.Token;

public class Controller extends Base implements Greeter {
    private Service service;

    public Token issue(String input) {
        return new Token(input);
    }

    @Override
    public String greet(String user) {
        return service.greet(user);
    }
}